lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@jcoyne @njaffer OK, I worked on this a bit and wrote it up. See here:

https://github.com/projecthydra/sufia/pull/1540#issue-129226479

Note that adding `app/models/sufia` did **not** help at all.